pkg/rngd/cmd/rngd/go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/lf-edge/eve/pkg/rngd
+
+go 1.20
+
+require golang.org/x/sys v0.9.0
+
+//replace golang.org/x/sys => golang.org/x/sys e312636bdaa2fac4f0acde9d17ab9fbad2b4ad10
```

Comment:
The old `vendor.conf` had this specific commit. It certainly is much older than the `v0.9.0` it automatically pulled down. I kept it in here as a comment, just in case we run into any obscure issues and need to figure out which commit it was, as unlikely as that is.